manifests/3/34u/MultiBaseCSHome/5.1.1.182/34u.MultiBaseCSHome.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: 34u.MultiBaseCSHome
+PackageVersion: 5.1.1.182

```

```suggestion
PackageVersion: 4.9.0.99
```
This should match what the package writes to Control Panel. After changing this, the version folder will need to be renamed
```
--> Comparing ARP Entries

DisplayName        DisplayVersion Publisher ProductCode                            Scope
-----------        -------------- --------- -----------                            -----
MultiBaseCS 5 Home 4.9.0.99       34u GmbH  {4ECEB5BA-9374-4D5C-9F77-95FBBA24DE0F} Machine
```